docs/sources/configuration/_index.md

```diff
@@ -817,6 +817,12 @@ storage:
 # CLI flag: -ruler.rule-path
 [rule_path: <string> | default = "/rules"]
 
+# Upper bound of random duration to wait before rule evaluation; this is to
```

Comment:
nit: might be useful to say that this is "experimental" (in case you think it is good to avoid using it for now for very sensitive environments)

Reply:
Nope, I don't think there's any risk at all.
The same instant is used when evaluating the rule, only we wait slightly before evaluating